Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gcr.io/distroless/static
```

Comment:
Consider using this container image `gcr.io/distroless/static:nonroot`, so that the container does not run as root.